.gitignore

```diff
@@ -50,14 +53,27 @@ node_modules/
 bower_components/
 src/vendor
 
-# Sheer #
-#########
+# Vim swap files #
+##################
 *.swp
+
+# Python #
+#################
 *.pyc
+*.egg-info/
+__pycache__/
+*.py[cod]
+.env
+*.eggs/
+*.egg/
 
-# Built assets #
-################
-dist/
+# Front-End #
+#############
+node_modules/
+bower_components/
+.grunt/
+src/vendor/
+cfgov/v1/jinja2/v1/*
```

Comment:
yup just still be removed in the next pull request moving templates from src to this jinja2 location